internal/pkg/expanders/containerRegistry.go

```diff
@@ -678,7 +678,3 @@ func (e *ContainerRegistryExpander) getRegistryToken(ctx context.Context, loginS
 
 	return accessToken, nil
 }
-
-func (e *ContainerRegistryExpander) testCases() (bool, *[]expanderTestCase) {
```

Comment:
Nice tidying!

Reply:
Thanks - I felt a bit bad bundling it in this PR, but.... 😉